File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P01_ChooseDrink.java
import java.util.Scanner;
public class P01_ChooseDrink {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        String profession = console.nextLine().toLowerCase();

        switch(profession) {
            case "athlete":
                System.out.println("Water");
                break;
            case "businessman":
            case "businesswoman":
                System.out.println("Coffee");
                break;
            case "softuni student":
                System.out.println("Beer");
                break;
            default:
                System.out.println("Tea");
                break;
        }
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P01_ChooseDrink/P01_ChooseDrink.cs
using System;

namespace P01_ChooseDrink
{
    class P01_ChooseDrink
    {
        static void Main(string[] args)
        {
            string profession = Console.ReadLine().ToLower();
            switch (profession)
            {
                case "athlete":
                    Console.WriteLine("Water");
                    break;
                case "businessman":
                case "businesswoman":
                    Console.WriteLine("Coffee");
                    break;
                case "softuni student":
                    Console.WriteLine("Beer");
                    break;
                default:
                    Console.WriteLine("Tea");
                    break;
            }
        }   
    }
}
